plugins(acme) add document for external account binding (#2491)
Kong/docs.konghq.com: app/_hub/kong-inc/acme/index.md

      default: "`false`"
      description: |
        If you are using Let's Encrypt, you must set this to true to agree the [Terms of Service](https://letsencrypt.org/repository/).
    - name: eab_kid
      required: false
      description: |

>         External account binding (EAB) key id. User usually doesn't need to set this unless explictly required by CA.

Reviewer: ```suggestion
        External account binding (EAB) key id. You usually don't need to set this unless it is explicitly required by the CA.
```

    - name: eab_hmac_key
      required: false
      description: |

>         External account binding (EAB) base64 encoded URL string of the HMAC key. User usually doesn't need to set this unless explictly required by CA.

Reviewer: ```suggestion
        External account binding (EAB) base64-encoded URL string of the HMAC key. You usually don't need to set this unless it is explicitly required by the CA.
```

    To configure storage type other than `kong`, please refer to [lua-resty-acme](https://github.com/fffonion/lua-resty-acme#storage-adapters).

> Externel account binding (EAB) is supported as long as `eab_kid` and `eab_hmac_key` is provided.

Reviewer: Not sure if this is both or either (or)?

```suggestion
Externel account binding (EAB) is supported as long as `eab_kid` and `eab_hmac_key` are provided.
```

> The following CA provider's external account can be registered automatically and user doesn't need

Reviewer: ```suggestion
The following CA provider's external account can be registered automatically, without specifying
```

> to specify `eab_kid` or `eab_hmac_key`:

Reviewer: ```suggestion
the `eab_kid` or `eab_hmac_key`:
```
